### ISSUE TITLE
Make value box size parametrized

### ISSUE THREAD

**Issue by yorl1n** [Mikhus/canvas-gauges#70]
Now the width of value box is defined as follows:
`let tw = context.measureText(options.valueText ?
        text : ('-' + padValue(0, options))).width;`

So the width is defined according to initial value length.

F.e. I don't need leading zeroes and decimals (000.00)-> I specify component as follows: 
```
valueDec: 0,                                     
valueInt: 1,
```
Initial value is 0, the width of the value box is appropriate. Now I load value 1000 and it looks ugly. 
So I propose to move out dimensional config of the value box and make it configurable in px and % (from the total dimensions).

**Comment by Mikhus** [comment]
It makes sense, thank you. So I will keep the current behavior as default one and if there would be width specified it will be applied then.
This is scheduled for the next release 2.0.8.

**Comment by Mikhus** [comment]
Implemented in release 2.0.8, published.
